src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -143,10 +143,30 @@ public bool TryGetWellKnownAttribute(WellKnownAttributeFlags flag, out bool valu
             PEModuleSymbol moduleSymbol,
             PEMethodSymbol containingSymbol,
             int ordinal,
-            ParamInfo<TypeSymbol> parameter,
+            ParamInfo<TypeSymbol> parameterInfo,
+            bool isreturnParam,
             out bool isBad)
         {
-            return Create(moduleSymbol, containingSymbol, ordinal, parameter.IsByRef, parameter.RefCustomModifiers, parameter.Type, parameter.Handle, parameter.CustomModifiers, out isBad);
+            var parameter = Create(moduleSymbol, containingSymbol, ordinal, parameterInfo.IsByRef, parameterInfo.RefCustomModifiers, parameterInfo.Type, parameterInfo.Handle, parameterInfo.CustomModifiers, out isBad);
+            var inAttributeModreq = parameter.RefCustomModifiers.Any(modifier => !modifier.IsOptional && modifier.Modifier.IsWellKnownTypeInAttribute());
```

Comment:
>var [](start = 12, length = 3)

Please spell out the type.

Reply:
`InAttribute` is the type name. Did you mean leaving a comment?

---
In reply to: [166435944](https://github.com/dotnet/roslyn/pull/24524#discussion_r166435944) [](ancestors = 166435944)